odk1-src/central-users.rst

```diff
@@ -3,45 +3,92 @@
 Managing Users in Central
 =========================
 
-There are two types of user accounts in ODK Central: **web users** and **app users**.
-
- - **Web users** have accounts on the Central management website. These accounts are global across all projects on the server. They can log into the web interface and perform administrative actions like user management, form upload and management, and submission data viewing and download.
- - **App users** can use mobile data collection apps like ODK Collect to :ref:`connect to Central <central-users-app-configure>`. App Users are limited to a single project at a time. Once connected through the app, they will be able to see the list of forms, download the ones they need, and upload completed submissions to those forms.
-
-You will need both types of users in order to run a successful data collection project: a web user must upload a valid form definition, an app user must upload submissions to it from their mobile device, and the web user will then be able to see those submissions in the web interface and download or sync them for analysis.
+There are two types of user accounts in ODK Central: **Web Users** and **App Users**.
+
+ - **Web Users** have accounts on the Central management website. These accounts are global across all projects on the server. They can log into the web interface and perform administrative actions like user management, form upload and management, and submission data viewing and download.
+ - **App Users** can use mobile data collection apps like ODK Collect to :ref:`connect to Central <central-users-app-configure>`. App Users are limited to a single project at a time. Once connected through the app, they will be able to see the list of forms, download the ones they need, and upload completed submissions to those forms.
+
+You will need both types of users in order to run a successful data collection project: a Web User must upload a valid form definition, an App User must upload submissions to it from their mobile device, and the Web User will then be able to see those submissions in the web interface and download or sync them for analysis.
+
+.. _central-users-web-roles:
+
+Web User Roles
+--------------
+
+Central features Role-based User permissioning. In the current release of Central, we provide two predetermined roles: Administrator and Project Manager. In future releases, more default roles will be added and eventually you will be able to define your own roles as you see fit.
+
+By default, Central roles are configured to allow the following:
+
++------------------+---------------+-----------------+
```

Comment:
I like the idea of the table, but it's a little hard to follow. You simplify it by removing the table and saying project managers can't create projects, manage web users, or server configurations.

If you want to proceed with the table, I'd move Web Users down below Project Form Submissions because having it as the first row clashes with the Administrator/PM header. Another change to consider is trying to add some indent of the actions (but not their grouping). Something like this?

* Web Users
   * Create
* Projects
   * Create
   * Edit Details

Reply:
okay, i'll shuffle the rows. i think

1 when we add eg Project Viewer we'll be glad it's a table and
2 eg security auditors and IT staff are used to seeing this information in a matrix and having every verb enumerated just makes everybody feel more comfortable

Reply:
Understood the need for a table and shuffling has made things better. I still think it's a little hard to read. If the actions could be indented a bit, I'd be better, but if it's a lot of work, then punt.

Reply:
i tried to indent them and couldn’t figure out how to make it happen. if you have ideas i’m happy to try them.